sql/core/src/main/scala/org/apache/spark/sql/streaming/progress.scala

```diff
@@ -33,27 +34,6 @@ import org.apache.spark.sql.catalyst.util.DateTimeUtils
 
 /**
  * :: Experimental ::
- * Information about updates made to stateful operators in a [[StreamingQuery]] during a trigger.
```

Comment:
aah, then for consistency, SourceProgress and SinkProgress should also be before StreamingQueryProgress. But thats a bigger change should be done in a different PR.